another clause (#299)
metalbear-co/mirrord.dev: content/docs/managing-mirrord/security/index.md

    - `CAP_NET_ADMIN` - for modifying routing tables
    - `CAP_SYS_PTRACE` - for reading the target pod's environment variables
    - `CAP_SYS_ADMIN` - for joining the target pod's network namespace

> - mirrord doesn't copy/sync files/secrets (unless log to file is active in debug/trace, then some info might get dumped there) to the filesystem. The access it provides to secret/files is in memory (unless app itself does write/leak it).

Reviewer: ```suggestion
- mirrord doesn't copy remote files or secrets to the local filesystem. The local app only gets access to remote files and secrets is in memory, and so they'll only be written to the local filesystem if done by the local app.
```
